scan/src/reusable/InfoMobileCard.re

```diff
@@ -11,6 +11,7 @@ type t =
   | DataSource(ID.DataSource.t, string)
   | OracleScript(ID.OracleScript.t, string)
   | RequestID(ID.Request.t)
+  | ProgressBar(int, int, int)
```

Comment:
Let's declare these parameters as the record type. So, we can simply understand when we use them.